modules/gitops-release-notes-1-4-0.adoc

```diff
@@ -0,0 +1,52 @@
+// Module included in the following assembly:
+//
+// * gitops/gitops-release-notes.adoc
+
+[id="gitops-release-notes-1-4-0_{context}"]
+= Release notes for {gitops-title} 1.4.0
+
+{gitops-title} 1.4.0 is now available on {product-title} 4.7, 4.8, and 4.9.
+
+[id="new-features-1-4-0_{context}"]
+== New features
+
+In addition to the fixes and stability improvements, the following sections highlight what is new in {gitops-title} 1.4.0:
+
+* Upgraded {gitops-title} Application Manager (kam) to version *0.0.41*. link:https://issues.redhat.com/browse/GITOPS-1669[GITOPS-1669]
+
+* Upgraded Argo CD to version *2.2.2*. link:https://issues.redhat.com/browse/GITOPS-1532[GITOPS-1532]
+
+* Upgraded Helm to version *3.7.1*. link:https://issues.redhat.com/browse/GITOPS-1530[GITOPS-1530]
+
+* This enhancement adds the health status of `DeploymentConfig` resources, `Route` resources, and Operators that you install using OLM to the Argo CD Dashboard and {product-title} web console. This enables you to better monitor the overall health status of your application. link:https://issues.redhat.com/browse/GITOPS-655[GITOPS-655], link:https://issues.redhat.com/browse/GITOPS-915[GITOPS-915], link:https://issues.redhat.com/browse/GITOPS-916[GITOPS-916], link:https://issues.redhat.com/browse/GITOPS-1110[GITOPS-1110]
+
+* With this update, the Operator enables you to specify the number of desired replicas for the `argocd-server` and `argocd-repo-server` components by setting the `.spec.server.replicas` and `.spec.repo.replicas` attributes in the Argo CD custom resource, respectively. If the horizontal pod autoscaler (HPA) is configured for the `argocd-server` components, it takes precedence over the Argo CD custom resource attributes. link:https://issues.redhat.com/browse/GITOPS-1245[GITOPS-1245]
```

Comment:
For all instances of `With this update` shouldn't it be `With these updates`? You intro sentence says `fixes _and_ stability improvements` and you bullet list includes multiple upgrades, enhancements, and updates. So shouldn't they all be in plural?

Reply:
To avoid anthropomorphism, could you say, `With this update, you can use the Operator to specify the number of desired replicas...`

Reply:
> For all instances of `With this update` shouldn't it be `With these updates`? You intro sentence says `fixes _and_ stability improvements` and you bullet list includes multiple upgrades, enhancements, and updates. So shouldn't they all be in plural?

"With this update" is our standard phrasing [from the SSG](https://redhat-documentation.github.io/supplementary-style-guide/#release-notes). It is equivalent to "In the current release..."

Reply:
Good suggestions. Updated.